tests/integrationv2/test_well_known_endpoints.py

```diff
@@ -0,0 +1,65 @@
+import copy
+import os
+import pytest
+
+from configuration import available_ports, PROTOCOLS
+from common import ProviderOptions, Protocols, Ciphers
+from fixtures import managed_process
+from providers import Provider, S2N
+from utils import invalid_test_parameters, get_parameter_name
+
+
+ENDPOINTS = [
+    {"endpoint": "amazon.com"},
+    {"endpoint": "facebook.com"},
+    {"endpoint": "google.com"},
+    {"endpoint": "netflix.com"},
+    {"endpoint": "s3.amazonaws.com"},
+    {"endpoint": "twitter.com"},
+    {"endpoint": "wikipedia.org"},
```

Comment:
wikipedia often fails as they filter out our test requests. either make sure we receive a warning instead of a failure or remove it from the list

Reply:
I thought this was fixes in #1961 but it is still failing (just not for me on EC2). I'll add the expected failure list.

Reply:
It's one of the flaky tests